pkg/daemon/ceph/agent/flexvolume/server.go

```diff
@@ -71,7 +72,7 @@ func NewFlexvolumeServer(context *clusterd.Context, controller *Controller) *Fle
 func (s *FlexvolumeServer) Start(driverVendor, driverName string) error {
 	// first install the flexvolume driver
 	// /usr/local/bin/rookflex
```

Comment:
This comment is now kind of out of date. It seems best (but not necessary) to go ahead and remove it.

Reply:
done